includes/Media.php

```diff
@@ -30,12 +30,13 @@
  * Class Media
  */
 class Media {
+
 	/**
 	 * The image size for the poster-portrait-src.
 	 *
 	 * @var string
 	 */
-	const STORY_POSTER_IMAGE_SIZE = 'web-stories-poster-portrait';
+	protected $story_poster_image_size = 'web-stories-poster-portrait';
```

Comment:
This a little bit of performance overhead, converting to a property here, but I think the visiblity and ability to mock, is a fair tradeoff